src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -64,9 +64,9 @@
     public static final String VALID_SUBJECT_BOB = "mathematics";
     public static final String VALID_GRADE_AMY = "B+";
     public static final String VALID_GRADE_BOB = "A1";
-    public static final String VALID_EDUCATION_LEVEL_AMY = "secondary";
+    public static final String VALID_EDUCATION_LEVEL_AMY = "primary";
```

Comment:
I recommend not to change this part since these are used by everyone for testings. Are there any particular reason for this to be changed? It'd be better if you can create a local constant for your tests i.e : EDUCATION_LEVEL_PRIMARY